src/relay/op/tensor/reduce.cc

```diff
@@ -224,7 +238,19 @@ values over a given axis.
 )code" TVM_ADD_FILELINE)
 .set_attrs_type_key("relay.attrs.ReduceAttrs")
 .set_support_level(4)
-.add_type_rel("ArgReduce", ArgReduceRel);
+.add_type_rel("ArgReduce", ArgReduceRel)
+.set_attr<FTVMCompute>(
+  "FTVMCompute", [](const Attrs& attrs,
+                    const Array<Tensor>& inputs,
+                    const Type& out_type,
+                    const Target& target) {
+    const auto* param = attrs.as<ReduceAttrs>();
+    auto r_axes = GetReduceAxes(inputs[0]->shape.size(),
+                                param->axis, param->exclude);
```

Comment:
This is certainly the shortest line of addition to implement the feature. But not necessarily the fastest to run. Note that the reduction in topi already supports negative index. So we don't need to handle negative index here, the only thing we need to handle is exclude, we can use another function GetExcludeAxes to get the other axes and return them as Array<Integer> when there is exclude